src/Test/Integration/Model/Payment/FactoryTest.php

```diff
@@ -75,9 +76,15 @@ public function testPaymentMethodMapping($methodCode, $expectedPaymentMethod): v
     public function paymentMethodProvider(): array
     {
         return [
+            'return shopgate as default payment method' => ['SOMETHING', 'shopgate'],
+            'Bank Transfer'                             => ['PREPAY', 'banktransfer'],
+            'Cash on delivery'                          => ['COD', 'cashondelivery'],
+            'Invoice'                                   => ['INVOICE', 'checkmo'],
+            'Braintree Credit Card'                     => ['BRAINTR_CC', 'braintree'],
+            'Braintree PayPal'                          => ['BRAINTR_PP', 'braintree_paypal'],
             'return shopgate as default payment method' => ['PREPAY', 'shopgate'],
             'Braintree Credit Card'                     => ['BRAINTR_CC', 'braintree'],
-            'Braintree PayPal'                          => ['BRAINTR_PP', 'braintree_paypal']
+            'Braintree PayPal'                          => ['BRAINTR_PP', 'braintree_paypal'],
```

Comment:
@awesselburg please check ur warnings again, this should have popped up. Are they not showing for you?
<img width="839" alt="Screen Shot 2019-12-04 at 18 37 47" src="https://user-images.githubusercontent.com/844601/70156606-3ca47280-16c5-11ea-8034-0dccd9ea6f6b.png">